src/cardstack/al-cardstack-view.ts

```diff
@@ -373,25 +373,41 @@ export abstract class AlCardstackView< EntityType=any,
             if (this.characteristics.searchableBy.length === 0 ) {
                 return true;
             }
-            this.characteristics.searchableBy.find( (property:string) => {
-                if ( ! card.properties.hasOwnProperty( property ) || typeof( ( card.properties as any)[property] ) === 'undefined' ) {
-                    return false; //  terminate iteration
+            if(search.source.indexOf("=") !== -1){
```

Comment:
use .includes

Reply:
Includes is not supporting on IE11

Reply:
God bless polyfills...  definitely prefer `.includes()` for clarity.